src/eth/TaraClient.sol

```diff
@@ -202,8 +203,8 @@ contract TaraClient is IBridgeLightClient {
         internal
     {
         uint256 gasleftbefore = gasleft();
-        require(finalized.block.period + 1 == b.block.period, "Pending block should have number 1 greater than latest");
-        require(b.block.prevHash == finalized.blockHash, "Pending block must be child of latest");
+        require(finalized.block.period + 1 == b.block.period, "block.number != finalized.number +1");
```

Comment:
period + 1 ?

Reply:
Yeah, removed it everywhere, but for some reason forgot to do it here. Fixed